OpenRA.Mods.Common/Traits/World/OrderEffects.cs

```diff
@@ -0,0 +1,70 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
```

Comment:
2021

Reply:
I would suggest doing another PR correcting all of these. i copy pasted that header. Keeps happening with all my PR. So ill make a pr tomorrow to change all copyrights to 2021.